src/support.py

```diff
@@ -175,4 +175,12 @@ def tile_to_screen(pos):
 
 def screen_to_tile(pos):
     tile_size = TILE_SIZE * SCALE_FACTOR
-    return pos[0] // tile_size, pos[1] // tile_size
\ No newline at end of file
+    return pos[0] // tile_size, pos[1] // tile_size
+
+def interpolate_color(color1, color2, factor):
+    ''' interpolate / blend two colors '''
+    return (
+        int(max(0, min(255, color1[0] + (color2[0] - color1[0]) * factor))),
+        int(max(0, min(255, color1[1] + (color2[1] - color1[1]) * factor))),
+        int(max(0, min(255, color1[2] + (color2[2] - color1[2]) * factor))),
+    )
```

Comment:
You did not need to do that, as `pygame.Color` already implements a method for that, read the doc for it here: 
https://pyga.me/docs/ref/color.html#pygame.Color.lerp

Reply:
Hello I have added pygame.Color.lerp instead of that custom function, any other suggestion?